start_stop.lua

```diff
@@ -613,6 +602,8 @@ function courseplay:stop(self)
 		self.cp.driver:dismiss()
 	end
 
+	local ret2_removeLegacyCollisionTriggers = false
```

Comment:
why the variable?

Reply:
See next line - it gets the return value from the function call.
In general I think it is a good approach to get a feedback from a function if it was successful or not.
This could be used for further processing / error handling / information to the user...
I just want to make the developer aware that there is a result provided from the function, not only fire&forget

Reply:
see above